### DISCUSSION TITLE
How to create writable nodes from XML

### DISCUSSION THREAD

**Discussion by CasperAmandusJohansen** [FreeOpcUa/python-opcua#1334] [Q&A]
Hello, i am having some issues importing writeable nodes from XML.
Is this not implemented?

**Comment by swamper123** [comment]
The xml importer in this repo may be not working 100%, since this repo is in maintainance mode only.
You may want to try [opcua-asyncio](https://github.com/FreeOpcUa/opcua-asyncio), which is the most actual version of our python opc ua stack. 
If that one fails as well, you may want to create an issue in the asyncio repo (and I will see what I can do).
